### ISSUE TITLE
Upstream Kubernetes docs for Fedora need review/update

### ISSUE THREAD

**Issue by ncoghlan** [fedora-cloud/fedoracloud#1]
I was recently trying to get Kubernetes running on my laptop, and went through http://kubernetes.io/docs/getting-started-guides/fedora/fedora_manual_config/

I ended up with the system with a "Not Ready" node (which I haven't managed to debug yet), but noticed a couple of other problems:
- last checked version is given as Fedora 22 (and hence all the installation commands use `yum`)
- the title is "single node", but the fact it's separate node-and-master means it still isn't very good for running locally on your laptop without the overhead of a separate VM (at which point you may as well just use the ADB Vagrant VM), and hits many of the same networking complexities as multi-node
- I missed the suggestion of using ansible rather than doing it manually

**Comment by ncoghlan** [comment]
(While I originally thought there was a problem with the upstream instructions, I figured out today I _did_ have a bug in my config file changes, so kubelet wasn't starting, and hence the node was correctly being reported as not ready)

**Comment by jberkus** [comment]
So, main issue here is that it needs to be updated to Fedora23/24?

**Comment by ncoghlan** [comment]
Yeah, having finally debugged the problem with my own installation, I think that's the main thing. The other items that I found a bit confusing/surprising:
- explaining that "single node" means "master + single container node", not "single system"
- I landed on the page via a search link, and missed the ToC entry in the sidebar about potentially using Ansible instead

**Comment by jberkus** [comment]
Well, at this point you're getting into Kubernetes docs issues, not specifically Fedora.  That's how Kubernetes looks at "single node", it's kind of like European floor numbering ;-)

I'd love to hear your experience with doing Ansible, if you get around to testing that.

**Comment by ncoghlan** [comment]
Yeah, I think the only bit in scope for the Fedora Cloud team here is to keep an eye on the accuracy of the upstream Kubernetes docs - either by updating them directly, or moving them downstream into the Fedora Cloud documentation, and proposing replacing the upstream docs with pointers to the Fedora Cloud ones.

As far as trying out Ansible goes, I now have a working setup on my laptop based on a simple "k8s-all-in-one" service definition [1], so I'm unlikely to try any other "from scratch" setup methods any time soon.

[1] https://github.com/ncoghlan/container-utils/blob/master/services/k8s-all-in-one.service

**Comment by jberkus** [comment]
@ncoghlan yeah, I've wondered why we don't offer dev/demo examples using all-in-one.  Maybe I need to write one.

**Comment by trishnaguha** [comment]
@jberkus If you have not yet started writing the Doc, I can take up this issue :-).
